### PR TITLE
Add persistance to ory components

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**

Changes proposed in this pull request:

- Add database subchart to ory
- Enable persistance in hydra
- Create job for connection string creation

**Related issue(s)**

- Implements #6611

### CODE REVIEW

**Comment by Demonsthere** [comment]
During development https://github.com/ory/hydra-maester/issues/43 was discovered and reported

File: installation/resources/production/ory-persistance-profile.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```
Comment (piotrmsc) [inline installation/resources/production/ory-persistance-profile.yaml.tpl]: Production profile should be a single file in installation resources where all components will have its production profiles

File: installation/resources/production/ory-persistance-profile.yaml.tpl [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: ory
+    kyma-project.io/installation: ""
+data:
+  postgresql.enabled: "true"
+  hydra.hydra.autoMigrate: "true"
+  global.ory.hydra.persitance.enabled: "true"
+  # global.ory.hydra.persitance.user: "someUser"
```
Comment (piotrmsc) [inline installation/resources/production/ory-persistance-profile.yaml.tpl]: it looks the same as commented values in `values.yaml` chart. We have separate PR with docs that should describe and override for external DB.

**Pull request merged**